docs/Using-EthSigner/Getting-Started.md

```diff
@@ -31,6 +31,16 @@ option set to `8590`.
 
 Create a text file containing the password for the V3 Keystore key file to be created (for example, `passwordFile`). 
 
+!!! attention "The password text file must not contain end-of-line character"
+    Some POSIX compliant editors automatically add an end-of-line in text files but this must not be
+    the case in this file or EthSigner will consider this end-of-line as part of your password.
+    
+    Use the following command to safely make sure that the password file is correct :
+    ```bash
+    echo -n "Type your password:";read -s password;echo -ne $password > passwordFile;
```

Comment:
could this just be type "echo -ne _pasword_ > passwordfile" where _password_ is your literal password text, and passwordfile is the file which is to contain the password.

Reply:
This would disclose the password as the command would be recorded in the bash history

Reply:
ahhhh - interesting! 👍